src/Llvm.NET/Native/Generated.cs

```diff
@@ -866,13 +866,13 @@ internal static partial class NativeMethods
         internal static extern LLVMTypeRef LLVMDoubleType( );
 
         [DllImport( LibraryPath, EntryPoint = "LLVMX86FP80Type", CallingConvention = CallingConvention.Cdecl )]
-        internal static extern LLVMTypeRef LLVMX8FP80Type( );
+        internal static extern LLVMTypeRef LLVMX86FP80Type( );
 
         [DllImport( LibraryPath, EntryPoint = "LLVMFP128Type", CallingConvention = CallingConvention.Cdecl )]
-        internal static extern LLVMTypeRef LLVMFP18Type( );
+        internal static extern LLVMTypeRef LLVMFP128Type( );
 
         [DllImport( LibraryPath, EntryPoint = "LLVMPPCFP128Type", CallingConvention = CallingConvention.Cdecl )]
-        internal static extern LLVMTypeRef LLVMPPCFP18Type( );
+        internal static extern LLVMTypeRef LLVMPPCFP128Type( );
 
```

Comment:
Let's just delete the variants that don't take a context, as they aren't/shouldn't be used by the LLVM.NET projections. The default context is dangerous to use or assume in multi-threaded programming and creating a new context is as simple as `new Context()`. The P/Invoke calls are being moved directly into the classes using them so we'll be dropping all the others here as well as we go forward on that.